frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -88,6 +88,10 @@ func (c *Controller) AddRoutes(router *mux.Router) {
 		httpHandler(newExecCVDInstanceCommandHandler(c.Config, c.OperationManager, &powerwashCvdCommand{}))).Methods("POST")
 	router.Handle("/cvds/{group}/{name}/:powerbtn",
 		httpHandler(newExecCVDInstanceCommandHandler(c.Config, c.OperationManager, &powerbtnCvdCommand{}))).Methods("POST")
+	router.Handle("/cvds/{group}/{name}/:start_screen_recording",
+		httpHandler(newExecCVDInstanceCommandHandler(c.Config, c.OperationManager, &startScreenRecordingCvdCommand{}))).Methods("POST")
```

Comment:
The commit message says

> It's not possible to implement a truly RESTful api at the moment because the cvd command to start recording doesn't return the recording file name(s)

If it's not conformant anyway, can it the API least start with the HTTP paths we want, and the responses can be updated / made more complete later?

Reply:
I think that would be misleading: I would expect `POST /resource` to return something that identifies the created resource. Having the `:action` URL type instead indicates that an action was done on the device and it may not return much more than a success or failure flag.